### PR TITLE
redux impmlementation

### PR DESCRIPTION
**Pull request by dorkka**

### CODE REVIEW

File: src/components/ResortsData.js [hunk 1]
```diff
@@ -63,7 +57,7 @@ class ResortsData extends Component {
             <table className="table table-striped">
               <ResortsListHead />
               <tbody>
-                {data.map(resort => <ResortBasicInfo resort={resort} />)}
+                {map(data, (resort) => <ResortBasicInfo resort={resort} />)}
```
Comment (ar-mac) [inline src/components/ResortsData.js]: (minor)
We can use regular map, (not lodash one) as `data` is now array (not `byId` object).

File: src/components/ResortsData.js [hunk 2]
```diff
@@ -91,5 +85,26 @@ ResortsData.propTypes = {
...
+  setIsLoading: PropTypes.func.isRequired,
+  data: PropTypes.object.isRequired,
+  pageCount: PropTypes.number.isRequired,
+  isLoading: PropTypes.bool.isRequired,
+  error: PropTypes.object.isRequired,
 };
-export default ResortsData;
+
+const mapStateToProps = (state) => ({
+  data: getCurrentResorts(state),
```
Comment (ar-mac) [inline src/components/ResortsData.js]: Data can be renamed to something more descriptive like `resorts`

File: src/store/configureStore.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { createStore, combineReducers } from 'redux';
+import { composeWithDevTools } from 'redux-devtools-extension';
+import resorts from './resorts/reducer';
+
+const rootReducer = combineReducers({
+  resorts,
+});
+
+// const initialState = {};
```
Comment (pokiujf) [inline src/store/configureStore.js]: obsolete

**Review by ar-mac — COMMENTED** [review]
Minor changes. Overall looks very nice

**Review by ar-mac — APPROVED** [review]

**Pull request merged**